iceoryx2-bb/log/src/lib.rs

```diff
@@ -179,6 +188,27 @@ pub enum LogLevel {
     Fatal = 5,
 }
 
+impl FromStr for LogLevel {
+    type Err = ();
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
```

Comment:
I have wrote a part of it this afternoon, but you go ahead. Cool!

if it never returns an error, why not return LogLevel directly?